src/FSLibrary/MissionMaxTPSMixed.fs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module MissionMaxTPSMixed
+
+// This module provides a max TPS test with a blend of classic and Soroban load.
+// It uses the MaxTPSTest module to perform a binary search for the max TPS.
+
+open MaxTPSTest
+open MissionSimulatePubnet
+open StellarMissionContext
+open StellarCoreHTTP
+
+let maxTPSMixed (baseContext: MissionContext) =
+    let context =
+        { baseContext with
+              coreResources = SimulatePubnetTier1PerfResources
+              installNetworkDelay = Some(baseContext.installNetworkDelay |> Option.defaultValue true)
+              // Simulate apply duration using same distribution as
+              // `MissionSimulatePubnet`
+              simulateApplyDuration = Some(baseContext.simulateApplyDuration |> Option.defaultValue pubnetApplyDuration)
```

Comment:
A bit of context on `pubnetApplyDuration`: these numbers were sampled a few years ago and they only include classic transactions, so I think we need to refresh them for Soroban. Note that the distribution should match the settings in this mission `50% classic, 5% upload, 45% invoke`. Could we use operation application metrics from prod to come up with some reasonable values for each op type?

Reply:
Since soroban success rates are a little over 70% in my tests, I disabled this simulation entirely and just set a lower bound on the mandatory minimum success rate.